### Problema 1

La separación entre líneas del espectro rotacional puro de $\text{HCl}$ es $6,35×10^{11} \text{Hz}$. Determine la separación internuclear de esta molécula.

### Solución



La frecuencia de la separación entre líneas para un espectro rotacional puro esta dada por:

$$ \Delta (h \nu) = 2 B \qquad B = \frac{\hbar^2}{2\mu R_0^2} \qquad \mu = \frac{m_1 m_2}{m_1 + m_2} $$

$$ \Rightarrow R_0 = \sqrt{\frac{\hbar^2}{2 \mu B}} = \sqrt{\frac{\hbar^2}{\mu \Delta(h\nu)}}$$

Reemplazando:

- $\nu = 6,35×10^{11} \text{Hz}$
- $m_H \approx m_p \approx 1,6726 \times 10^{−27} \text{kg}$
- $m_{Cl} \approx 35.5 m_H$

In [13]:
from math import sqrt, pi
nu = 6.35e11
mH = 1.6726e-27
mCl = 35.5*mH
h =  6.626e-34
hbar = 1.054e-34
mu = lambda m1,m2: m1*m2/(m1+m2)
R0 = lambda nu,m1,m2: sqrt(hbar/(mu(m1,m2)*2*pi*nu))
result = R0(nu,mH,mCl)
print(result * 10**10, 'A')


1.2743235763373386 A


Obteniéndose que la separación inter nuclear de la molécula de $\text{HCl}$ es de $1.27 \text{Å}$.

### Problema 2. 
Calcule la energía **rotacional** de los tres primeros niveles rotacionales puros molécula
$\text{CO}$. Determine la longitud de onda de los fotones absorbidos en las transiciones entre
estos niveles de energía. La longitud de enlace de la molécula es $1,13\text{Å}$

### Solución

Los niveles de energía rotacionales de una molécula diatómica son:

$$ E_{rotacional}^l = \frac{\hbar^2l(l+1)}{2\mu R_0^2} \qquad l = 1,2,... \qquad \mu = \frac{m_1 m_2}{m_1 + m_2} $$

Reemplazando:

- $R_0 = 1,13\text{Å}$
- $m_H \approx m_p \approx 1,6726 \times 10^{−27} \text{kg}$
- $m_C \approx 12 m_H$
- $m_O \approx 16 m_H$


In [9]:
mH = 1.6726e-27
mC = 12*mH
mO = 16*mH
hbar = 1.054e-34
R0 = 1.13e-10

mu = lambda m1,m2: m1*m2/(m1+m2)
E = lambda l, R0, m1, m2: hbar**2*l*(l+1)/(2*mu(m1,m2)*R0**2)

for l in range(1,4):
    print(E(l,R0,mC,mO)/1.6e-19*1000, 'meV')

0.47409902515390406 meV
1.4222970754617121 meV
2.8445941509234243 meV


Los primeros tres niveles de energía rotacional son:

$$ E_{rotacional}^1 = 0.46 \text{meV} \qquad E_{rotacional}^2 = 1.42 \text{meV} \qquad E_{rotacional}^3 = 2.84 \text{meV} $$

Por otra parte la longitud de onda de los fotones absorbidos en las transiciones entre estos niveles de energía es:

$$ \lambda = \frac{hc}{\Delta E} = \frac{hc}{2B} \qquad B = \frac{\hbar^2}{2\mu R_0^2} $$

In [14]:
mH = 1.6726e-27
mC = 12*mH
mO = 16*mH
hbar = 1.054e-34
R0 = 1.13e-10
h =  6.626e-34
c = 3e8
mu = lambda m1,m2: m1*m2/(m1+m2)
B = lambda m1, m2, R0: hbar**2/(2*mu(m1,m2)*R0**2)
wavelenght = lambda m1, m2, R0: h*c/(2*B(m1,m2,R0)) 

print(wavelenght(mC,mO,R0)*10**3, 'mm')

2.620496845773296 mm


Obteniéndose una longitud de onda de absorción de $ \lambda = 2.6 \text{mm} $, dicha longitud de onda corresponde al espectro de microondas.



### Problema 3. 

Una molécula diatómica no es rigurosamente rígida, por lo que la distancia internuclear aumenta a medida que aumenta su momento angular.

Analizar cómo afecta este estiramiento a los niveles de energía rotacional.

Una expresión empírica para la energía rotacional, considerando el efecto de distorsión centrífuga, es:

$$ E_l = B_0 l(l+1) - D l^2(l+1)^2 $$

Encontrar una expresión para la constante de distorsión centrífuga $D$. Considere un modelo en el que la fuerza de lugadura obedece la ley de Hook con una constante de fuerza $k$ y que los desplazamientos respecto a la separación de equilibrio son pequeños.

Encontrar una expresión para las frecuencias correspondientes a las transiciones entre los niveles rotacionales.

### Solución

Supongamos que la fuerza de la ligadura obedece la ley de Hook, es decir, dada una constante de fuerza $k$, y suponiendo que los desplazamientos con respecto a la posición de equilibrio son pequeños. La molecula experimentara una fuerza $F = k(R - R_0)$ al alejarse de la posición de equilibrio. Si igualamos esta fuerza con la fuerza centripeta $F = \mu \omega^2 R$  Para estiramientos pequeños:

$$ \Rightarrow k(R - R_0) \approx \mu \omega^2 R_0 $$

Reemplazando en la expresión de la energía rotacional:

$$ E_{rotacional}^l = Bl(l+1) + \frac{1}{2} k(R - R_0)^2 \approx \frac {\hbar^2 l(l+1)}{2 \mu R_0^2} + \frac{\omega^4 R_0^2 \mu^2}{2k} $$

De la ecuación del momento angular, dado un momento de inercia $I = \mu R_0^2$:

$$ \hat{L} = I \hat{\omega} = \mu R_0^2 \hat{\omega} \Rightarrow \frac{\omega^4 R_0^2 \mu^2}{2k} = \frac{R_0^2 \mu^2}{2k} \frac{\hat{L}^2}{\mu^4 R_0^8}$$

Reemplazando en la expresión los operadores por sus autovalores, obtenemos la energia:

$$E_{rotacional}^l = Bl(l+1) + \frac{1}{2}k(R - R_0)^2 \approx \frac{\hbar^2 l(l+1)}{2 \mu R_0^2} + \frac{\hbar^4l^2(l+1)^2}{2k\mu^2 R_0^6}$$


En donde $D = \frac{\hbar^4}{2k\mu^2 R_0^6}$


### Problema 4

Evaluar la energía de punto cero para la molécula de $\text{Ar}_2$ a partir del potencial de Lennard-Jones.
Repetir el cálculo anterior para la molécula de $\text{He}_2$. A partir de estos resultados discutir la posibilidad de formarse la molécula de $\text{He}_2$.

Los parametros del potencial de Lennar-Jones para $\text{Ar}$ y $\text{He}$ son $\sigma = 0.340 \text{nm}$, $\epsilon = 10.5 \text{meV}$ y $\sigma = 0.256 \text{nm}$, $\epsilon = 0.879 \text{meV}$ respectivamente.

### Solución

El potencial de Leannard-Jones está dado por:

$$ V_{LJ}(R) = 4 \epsilon \left[ \left( \frac{\sigma}{R} \right)^{12} - \left( \frac{\sigma}{R} \right)^{6} \right]$$

Las derivadas del potencial de Lennard-Jones son:


$$ V_{LJ}'(R) = 24 \epsilon_0 \left( -2 \frac{\sigma^{12}}{R^{13}} + \frac{\sigma^6}{R^7} \right) \qquad V_{LJ}''(R) = 24 \epsilon \left( 26\frac{\sigma^{12}}{R^{14}} - 7\frac{\sigma^6}{R^8} \right) $$

Podemos hallar de esta forma, la separación internuclear de equilibrio $R_0$, tomando $V_{LJ}'(R_0) = 0$:

$$ 24 \epsilon \left( -2 \frac{\sigma^{12}}{R_0^{13}} + \frac{\sigma^6}{R_0^7} \right) = 0$$

$$ -2 \frac{\sigma^{6}}{R_0^{6}} + 1 = 0 \Rightarrow R_0 = 2^{1/6}\sigma$$

Tomando $k = V''(R_0)$, podemos aproximar el potencial de Lennard-Jones alrededor de $R_0$ como un oscilador armónico:

$$ V(R) \approx -\epsilon + \frac{1}{2}k(R - R_0)^2 \qquad k = \omega^2 \mu$$ 

Evaluando:

$$ k = V''(R_0) = 24 \epsilon \left( 26\frac{\sigma^{12}}{(2^{1/6}\sigma)^{14}} - 7\frac{\sigma^6}{(2^{1/6}\sigma)^8} \right)$$

$$ = 24 \epsilon \left( \frac{26}{2^{14/6} \sigma^2} - \frac{7}{2^{8/6}\sigma^2} \right) = \frac{24 \epsilon}{\sigma^2} (\frac{2^{14/6} 13 - 2^{14/6} 7}{2^{22/6}}) = \frac{24 \epsilon}{\sigma^2} \frac{2^{14/6}}{2^{22/6}} (13 - 7) = \frac{72 \epsilon}{2^{1/3} \sigma^2}$$

$$ \Rightarrow V(R) \approx -\epsilon + \frac{36 \epsilon}{2^{1/3} \sigma^2} (R - 2^{1/6}\sigma)^2 $$

Reemplazando en $k = \omega^2 \mu$, podemos obtener la frecuencia de oscilación del espectro vibracional para una molecula diatomica:

$$ \omega = \sqrt{\frac{72\epsilon}{2^{1/3} \sigma^2 \mu}} \qquad \mu = \frac{m_1 m_2}{m_1 + m_2}$$

Podemos entonces obtener la energía de punto cero para moléculas diatomicas como el $\text{Ar}_2$ y $\text{He}_2$ como:

$$ E_0 = \frac{1}{2} \hbar \omega = \hbar \sqrt{\frac{18 \epsilon}{2^{1/3} \sigma^2 \mu}}$$

Reemplazando para el $\text{Ar}_2$:

- $\sigma = 0.340 \text{nm}$
- $\epsilon = 10.5 \text{meV}$
- $m_{Ar} \approx 40 m_H$

In [10]:
from math import sqrt
sigma = 0.340e-9 # m
epsilon = 10.5e-3 # eV
mH = 1.6726e-27
mAr = 40*mH # kg
hbar = 1.054e-34 # J s

def eV_to_J(energy):
    return energy*1.609e-19

def J_to_eV(energy):
    return energy/1.609e-19

mu = lambda m1,m2: m1*m2/(m1+m2)
E = lambda m1, m2, sigma, epsilon: hbar*sqrt(18*epsilon/(2**(1/3)*sigma**2*mu(m1,m2)))

print(J_to_eV(E(mAr,mAr,sigma, eV_to_J(epsilon))) * 1000, 'meV')


1.6365597426795047 meV


Reemplazando para el $\text{He}_2$:
- $\sigma = 0.256 \text{nm}$
- $\epsilon = 0.879 \text{meV}$
- $m_{He} \approx 4 m_H$

In [14]:
from math import sqrt
sigma = 0.256e-9 # m
epsilon = 0.879e-3 # eV
mH = 1.6726e-27
mHe = 4*mH # kg
hbar = 1.054e-34 # J s

def eV_to_J(energy):
    return energy*1.609e-19

def J_to_eV(energy):
    return energy/1.609e-19

mu = lambda m1,m2: m1*m2/(m1+m2)
E = lambda m1, m2, sigma, epsilon: hbar*sqrt(18*epsilon/(2**(1/3)*sigma**2*mu(m1,m2)))

print(J_to_eV(E(mAr,mAr,sigma, eV_to_J(epsilon))) * 1000 , 'meV')

0.6288840839995207 meV


Las energías de punto cero para el $\text{Ar}_2$ y $\text{He}_2$ son de $1.63 \text {meV}$ y $0.62 \text{meV}$ respectivamente. La temperatura de la energía térmica equivalente será:

$$ k_B T = E_0 \Rightarrow T = \frac{E_0}{k_B}$$



In [11]:
kB = 8.617333262145 # eV/K
T = lambda E0: E0/kB
E0 = 1.63e-3 # eV
print(T(E0), 'K')
E0 = 0.62
print(T(E0), 'K')

0.00018915364538127025 K
0.07194801235361199 K


Las temperaturas del punto cero para el $\text{Ar}_2$ y $\text{He}_2$ son de casi $0K$, lo que indica que estas moléculas no existen en la naturaleza.

### Problema 5

Para muchas moléculas ligadas covalentemente, la energía potencial se puede describir adecuadamente por medio del potencial de Morse:

$$ V(R) = A \left( e^{-2a(R-R_0)} - 2e^{-a(R-R_0)} \right) $$

Donde $A$, $a$ y $R_0$ son parámetros característicos de cada molécula. La separación internuclear de la molécula $\text{CO}$ es $0.113 \text{nm}$, la energía de disociación es de $9.60 \text{eV}$, la frecuencia vibracional es de $6.51 \times 10^{13} \text{Hz}$. A partir de estos datos experimentales, determinar los potenciales de Morse para la interacción entre el carbono y el oxígeno.

### Solución

La energía de disociación de una molécula diatómica es la energía necesaria para separar los átomos a una distancia infinita, es decir:

$$ \Delta E = V(\infty) - V(R_0) = 0 - V(R_0) = A = 9.60 \text{eV}$$

Hallemos ahora el parámetro $a$. Derivamos el potencial de Morse:

$$ V'(R) = A \left( -2a e^{-2a(R-R_0)} + 2ae^{-a(R-R_0)} \right) \qquad V''(R) = A \left( 4a^2 e^{-2a(R-R_0)} - 2a^2 e^{-a(R-R_0)} \right) $$

Es inmediato que $V'(R_0) = 0$. Desarrollando a segundo orden alrededor de $R_0$, podemos aproximar el potencial de Morse como un oscilador armónico:

$$ V(R) \approx -\epsilon + \frac{1}{2} k(R - R_0)^2 \qquad k = \omega^2 \mu $$

En donde:

$$ -\epsilon = V(R_0) = -A \qquad k = V''(R_0) = A \left( 4a^2 e^{-2a(R_0-R_0)} - 2a^2 e^{-a(R_0-R_0)} \right) = 2Aa^2 $$

Reemplazando en $k = \omega^2 \mu$:

$$ \omega^2 \mu = 2\epsilon a^2 \Rightarrow a = \sqrt{\frac{\omega^2 \mu}{2\epsilon}} $$

En donde habiamos encontrado que $\epsilon$ es la energía de disociación. Reemplazando:

- $\epsilon = 9.60 \text{eV}$
- $\nu = \omega / 2\pi = 6.51 \times 10^{13} \text{Hz}$

In [22]:
from math import pi, sqrt

epsilon = 9.6 # eV
nu = 6.51e13 # Hz
mH = 1.6726e-27 # kg
mO = 16*mH
mC = 12*mH

def eV_to_J(energy):
    return energy*1.609e-19

def J_to_eV(energy):
    return energy/1.609e-19

omega = lambda nu: 2*pi*nu
mu = lambda m1,m2: m1*m2/(m1+m2)
a = lambda epsilon, omega, mu: sqrt(omega**2*mu/(2*epsilon)) 

print(a(eV_to_J(epsilon), omega(nu), mu(mC,mO))*10**-10, 'e^10[1/m]')

2.4922974057924216 e^10[1/m]
